crates/bevy_sprite/src/mesh2d/material.rs

```diff
@@ -78,7 +89,13 @@ impl<M: Material2d> SpecializedMaterial2d for M {
     fn key(_material: &<Self as RenderAsset>::PreparedAsset) -> Self::Key {}
 
     #[inline]
-    fn specialize(_key: Self::Key, _descriptor: &mut RenderPipelineDescriptor) {}
+    fn specialize(
+        _key: Self::Key,
+        _descriptor: &mut RenderPipelineDescriptor,
+        _layout: &MeshVertexBufferLayout,
```

Comment:
Why _ prefixes for used parameters?

Reply:
Ah yeah it used to be an "empty" impl and I didn't update this. Nice catch!